### ISSUE TITLE
Debugging an aspnetcore library targeting netcoreapp3.0

### ISSUE THREAD

**Issue by TulyOpt** [dotnet/sourcelink#517]
Hello,

according to your readme : 
>  Microsoft libraries, such as .NET Core and Roslyn have enabled Source Link. Source Link is supported by Microsoft.

Do you confirm that I could debug an AspNetCore library targeting netcoreapp3.0 ? Because I can't even find the pdb in the nupkg.

**Comment by tmat** [comment]
Yes. The PDBs are not included in the packages. They are published on Microsoft Symbol Servers.

**Comment by TulyOpt** [comment]
OK, thanks for your answer. Does it work for linux ? And should I change my configuration to point to the Microsoft Symbol Servers ? What I can find for Microsoft Symbol Servers seems a bit outdated.
